src/app/public/modules/dropdown/dropdown-menu.component.ts

```diff
@@ -43,15 +43,37 @@ let nextId = 0;
 })
 export class SkyDropdownMenuComponent implements AfterContentInit, OnDestroy {
 
+  /**
+   * Sets the dropdown menu's `aria-labelledby` attribute to support accessibility. The value should
+   * be the HTML element ID (without the leading `#`) of the element that labels the dropdown menu.
+   * This property accepts `string` values.
```

Comment:
These types of sentences are probably unnecessary now that we're showing the type in the tables. @blackbaud-johnly What do you think?

Reply:
Awaiting @blackbaud-johnly 's response here, all other issues have been resolved

Reply:
Sorry for the slow response here. I haven't served this to check the docs, but I agree with Steve that the description shouldn't call out the type if we show the type in a separate column.